frontend/app/src/components/dashboard/DashboardExpandableTable.vue

```diff
@@ -5,35 +5,34 @@ const panel = computed<number>(() => (get(expanded) ? 0 : -1));
 </script>
 
 <template>
-  <Card :class="{ 'pb-6': expanded }">
-    <template #title>
-      <div class="flex items-center">
-        <div class="mr-2">
+  <RuiCard :class="{ '[&>div:last-child]:!py-0': !expanded }">
+    <template #custom-header>
+      <div class="flex justify-between items-center p-2 pr-4">
+        <CardTitle>
```

Comment:
Do you think it would make sense to use the RuiCardHeader here?

Reply:
I tried, and RuiCardHeader is not effective in this case, I ended up writing more tags there

Reply:
oh ok